libs/utils/src/lib/form/validators/validators.ts

```diff
@@ -43,7 +43,7 @@ export function confirmPasswords(
 /** Require current and new password to be different */
 export function differentPassword(current: string = 'current', next: string = 'next'): ValidatorFn {
   return (group: FormGroup): { [key: string]: boolean } | null => {
-    return group.controls[current].value === group.controls[next].value
+    return group.controls[current].value === group.controls[next].value && !group.controls[current].disabled && !group.controls[next].disabled
```

Comment:
it's not readable, please create intermediary variables : 
```suggestion
    const currentControl = group.controls[current];
    const nextControl = group.controls[next];
    if (currentControl !== nextControl.value) return null;
    if (currentControl.disabled && nextControl.disabled) return null;
    return { currentPasswordMatch: true };
```